tox.ini

```diff
@@ -17,8 +17,11 @@ deps =
     pytest-cov>=2.10,<3
     pytest-xdist>=2.1,<3
     hypothesis>=6.9,<6.10
+extras =
+    mypy: mypy
```

Comment:
Any specific reason for extras here? Let's make it and explicit dependency!

Reply:
Or alternatively, move mypy testing into a dedicated env, as we have done for wsproto https://github.com/python-hyper/wsproto/blob/55bf7a4ee1e85e3ebc93f04708327b0d0f193925/tox.ini#L40-L46

Reply:
It's convenient to use extras when developing outside of tox, I'm planning on moving the test deps to extras too